src/content/reference/domain-whitelist/index.md

```diff
@@ -0,0 +1,43 @@
+---
+linkTitle: Domain whitelist
+title: Domain whitelist
+description: A list of all external domains Giant Swarm clusters need access to.
+weight: 10
+menu:
+  main:
+    parent: reference
+user_questions:
+  - How is the Linux kernel of cluster nodes configured?
+owner:
+  - https://github.com/orgs/giantswarm/teams/se
+---
+
+# External Domains
+
+Below is a list of the external domains we require access to for our clusters to function.
+
+- flatcar.com
+    - *.flatcar.com
+- quay.io
+    - *.quay.io
+- github.com
+    - *.github.com
+- release.core-os.net
+    - *.release.core-os.net
```

Comment:
Why do we need this one?

Reply:
hmm that's a good point - i'm not sure we do. i was just dumping a list from a github issue. i'll remove it.